Dockerfile.sidecar

```diff
@@ -52,7 +52,7 @@ RUN useradd -u 999 -r -g 999 -s /sbin/nologin \
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
-        apt-transport-https ca-certificates wget \
+        apt-transport-https ca-certificates pigz wget \
```

Comment:
Please add this in `hack/development/Dockerfile.sidecar` too, sorry for this code duplication..

Reply:
Done: https://github.com/presslabs/mysql-operator/pull/624/commits/a342b29833739741d9401c40dacd01868cd3e32b